pkg/controllers/clusterresource/verifier_controller.go

```diff
@@ -70,12 +70,13 @@ func (r *VerifierReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	}
 
 	if err := verifierAddOrReplace(verifier.Spec, resource); err != nil {
```

Comment:
it looks like verifierAddOrReplace already returns "re.ErrorCodePluginInitFailure.WithError(err).WithDetail("Unable to create verifier from verifier crd.")". wondering this message be duplicated?

Reply:
Thanks for catching! I'll just return err here.

Reply:
I'll let `verifierAddOrReplace` just returns `err` and add details in `Reconcile` since `verifierAddOrReplace` could return 2 different errors.